google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurable.java

```diff
@@ -86,16 +89,19 @@ public boolean isModified() {
   }
 
   /**
-   * User's shouldn't be able to save a cloud configuration without a valid Cloud SDK configured.
+   * Users shouldn't be able to save a cloud configuration without a valid Cloud SDK configured.
    */
   @Override
   public void apply() throws ConfigurationException {
-    if (StringUtil.isEmpty(cloudSdkPanel.getCloudSdkDirectory())
-        || !CloudSdkUtil.containsCloudSdkExecutable(cloudSdkPanel.getCloudSdkDirectory())) {
+    try {
+      new CloudSdk.Builder()
+          .sdkPath(Paths.get(cloudSdkPanel.getCloudSdkDirectory()))
+          .build()
+          .validateCloudSdk();
```

Comment:
sorry I was wrong here - it happens when the user attempts to execute gcloud or appcfg, not on creating a new SDK like I said. Still it feels wrong to create a new one just for validation purposes